--- a/qtbase/src/plugins/bearer/bearer.pro      2016-12-01 11:17:04.000000000 +0300
+++ b/qtbase/src/plugins/bearer/bearer.pro      2021-03-29 20:57:16.000000000 +0300
@@ -8,7 +8,7 @@
 #win32:SUBDIRS += nla
 win32:SUBDIRS += generic
 win32:!wince:!winrt: SUBDIRS += nativewifi
-mac:contains(QT_CONFIG, corewlan):SUBDIRS += corewlan
+#mac:contains(QT_CONFIG, corewlan):SUBDIRS += corewlan
 mac:SUBDIRS += generic
 android:SUBDIRS += android
