diff -Nur openssl-0.9.8o/util/perlpath.pl openssl-0.9.8o.new/util/perlpath.pl
--- openssl-0.9.8o/util/perlpath.pl	2010-06-14 10:17:46.000000000 -0400
+++ openssl-0.9.8o.new/util/perlpath.pl	2010-06-14 10:18:04.000000000 -0400
@@ -11,6 +11,10 @@
 
 sub wanted
 	{
+	if (/^\.pc/) {
+		$prune = 1;
+		return;
+	}
 	return unless /\.pl$/ || /^[Cc]onfigur/;
 
 	open(IN,"<$_") || die "unable to open $dir/$_:$!\n";
